docs/structured-streaming-kafka-integration.md

```diff
@@ -414,6 +414,16 @@ The following configurations are optional:
   issues, set the Kafka consumer session timeout (by setting option "kafka.session.timeout.ms") to
   be very small. When this is set, option "groupIdPrefix" will be ignored. </td>
 </tr>
+<tr>
+  <td>setCommitOffsetsOnCheckpoints</td>
```

Comment:
Maybe `commitOffsetsOnCheckpoints` since this is not a function.

Reply:
ok.